test/e2e/node/pod_resize.go

```diff
@@ -110,9 +110,9 @@ func doPodResizeAdmissionPluginsTests() {
 	}
 
 	for _, tc := range testcases {
-		ginkgo.It(tc.name, func(ctx context.Context) {
-			f := framework.NewDefaultFramework(tc.name)
```

Comment:
This is not a mistake that can be found by the existing linters for E2E suites. The test has to run, then Ginkgo detects the mistake.

This begs the question: how was this test developed? Was it executed at all before merging https://github.com/kubernetes/kubernetes/pull/128266?

Reply:
It was executed and failed but for some reason did not block the PR from merging - https://prow.k8s.io/pr-history/?org=kubernetes&repo=kubernetes&pr=128266

Reply:
we don't block on flake tests, blocking presubmits jobs need to be very reliable and fast, is on reviewers authors to ensure their code is fine to merge. It is a trade off we want to have and stable, reliable and performance CI system

Reply:
I agree that they are non-blocking and optional jobs. But it should at least inform failures on GitHub. In my case, the job was triggered but did not appear under GitHub checks.

Reply:
the job config has a boolean to indicate to report status,. I didn;t check but maybe that was happened, we should switch that bool to on so people can look into the failures

Thanks for doing the retrospective with us